src/OpenTelemetry/Trace/BatchingActivityProcessor.cs

```diff
@@ -202,26 +202,13 @@ public override async Task ForceFlushAsync(CancellationToken cancellationToken)
             OpenTelemetrySdkEventSource.Log.ForceFlushCompleted(this.currentQueueSize);
         }
 
-        /// <inheritdoc/>
-        public void Dispose()
-        {
-            this.Dispose(true);
-        }
-
         /// <summary>
         /// Releases the unmanaged resources used by this class and optionally releases the managed resources.
         /// </summary>
         /// <param name="disposing"><see langword="true"/> to release both managed and unmanaged resources; <see langword="false"/> to release only unmanaged resources.</param>
-        protected virtual void Dispose(bool disposing)
+        protected override void Dispose(bool disposing)
```

Comment:
should be able to drop IDisposable if you added it to the base class...

Reply:
nice catch. thank you. i will review all processors and remove the idisposable!